### PR TITLE
Adding audit log event support

### PR DESCRIPTION
**Pull request by noxora**
I'd like to add audit log event support. I think I will still need to tag each type of carbonblack event to make it so one can key off it in logstash.

### CODE REVIEW

File: carbonclient/client.go [hunk 1]
```diff
@@ -86,5 +88,109 @@ func (c *Client) FetchNotifications() (Notifications, error) {
...
+		logp.Warn("500: Internal server error. retrying in a bit")
+		time.Sleep(5 * time.Minute)
+	default:
+		return events, fmt.Errorf("status code: %v - I don't know what this error "+
+			"is, it was not documented", code)
+	}
+	return events, nil
 }
+
+//	// FetchNotifications returns a batch of messages since t, time.Location
```
Comment (noxora) [inline carbonclient/client.go]: This can be removed but I decided to leave it here for reference if needed.

File: carbonclient/requests.go [hunk 2]
```diff
@@ -17,18 +17,45 @@ func parseNotificationsResBody(body []byte) (Notifications, error) {
...
 	}
 	req.Host = c.cfg.APIHost
-	req.Header.Set("X-Auth-Token", c.cfg.APIKey+"/"+c.cfg.ConnectorID)
+	req.Header.Set("X-Auth-Token", c.cfg.APIKey+"/"+c.cfg.APIConnectorID)
 	req.Header.Set("User-Agent", "Carbonbeat")
 	resp, err := c.httpClient.Do(req)
 	if err != nil {
 		logp.Err(err.Error())
-		time.Sleep(3 * time.Minute)
+		time.Sleep(15 * time.Minute)
```
Comment (noxora) [inline carbonclient/requests.go]: I didn't mean to change this, I'll correct this before merging

**Comment by Popsiclestick** [comment]
The changes LGTM. They have been tested and are functioning. 

Can you please add additions to the README to reflect the changes being made. We can merge and increment the version after that.

**Comment by noxora** [comment]
> The changes LGTM. They have been tested and are functioning.
> 
> Can you please add additions to the README to reflect the changes being made. We can merge and increment the version after that.

All Set on this. When we merge we should also squash the changes into one commit

**Pull request merged**